androidsvg/src/main/java/com/caverock/androidsvg/SVGMapImageView.java

```diff
@@ -0,0 +1,49 @@
+package com.caverock.androidsvg;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.util.AttributeSet;
+import android.widget.ImageView;
+
+import org.jetbrains.annotations.Nullable;
+
+
+public class SVGMapImageView extends SVGImageView {
```

Comment:
Let's name this something more generic if since we're intending for this to be usable by other perhaps?